thorlcr/slave/thslavemain.cpp

```diff
@@ -296,6 +296,15 @@ void setSlaveAffinity(unsigned processOnNode)
 
 int main( int argc, char *argv[]  )
 {
+    for (unsigned i=0;i<(unsigned)argc;i++) {
+        if (strcmp("--daemon",argv[i])==0 || strcmp("-d",argv[i])==0) {
+            if (daemon(1,0) || write_pidfile(argv[++i])) {
+                perror("Failed to daemonize");
+                return EXIT_FAILURE;
+            }
+            break;
+        }
```

Comment:
I'm not sure a slave can ever be a daemon (or at least not without other big changes). Slaves are surrogates to thormaster, launched and managed by it.
So I think this code should be remove from thslavemain.cpp